src/algebra/ordered_group.lean

```diff
@@ -700,9 +700,20 @@ le_of_forall_le_of_dense $ λ c hc,
 calc a ≤ b + (c - b) : h _ (sub_pos_of_lt hc)
    ... = c           : add_sub_cancel'_right _ _
 
+lemma le_iff_forall_pos_le_add [densely_ordered α] {x y : α} : x ≤ y ↔ ∀ ε > 0, x ≤ y + ε :=
```

Comment:
```suggestion
lemma le_iff_forall_pos_le_add [densely_ordered α] : a ≤ b ↔ ∀ ε, 0 < ε → a ≤ b + ε :=
```
for naming consistency with the lemmata around this, and since mathlib uses `<` rather than `>`, and similarly below